atc/db/pipeline.go

```diff
@@ -1147,26 +1147,39 @@ func (p *pipeline) Variables(logger lager.Logger, globalSecrets creds.Secrets, v
 }
 
 func (p *pipeline) SetParentIDs(jobID, buildID int) error {
+	if jobID <= 0 || buildID <= 0 {
+		return errors.New("job and build id cannot be zero-value")
```

Comment:
Mismatch between this error and the test (test expects `job and build id cannot be negative or zero-value`)

Reply:
weird, thought I changed that!